injective_functions/exchange/trader.py

```diff
@@ -150,8 +156,7 @@ async def place_spot_market_order(
     async def cancel_spot_limit_order(
         self, market_id: str, subaccount_idx: int, order_hash: str
     ):
-        await self.chain_client.init_client()
-
+        order_hash = base64convert(order_hash)
```

Comment:
Do not rewrite variables. Suggest using new names for new variables, will make this easier to read: 
```
converted_order_hash = base64.. 
resolved_market_id = await.. 
```